### PR TITLE
Upload IOS Travis CI

### PR DESCRIPTION
**Pull request by wrmswindmill**
if `WeexDemo` build success and  pass all the test case in WeexUITestDemo, then it will be success.
but there are not any test cases at all(only the test case template now!)

the following is the shell command:
```shell 
xcodebuild  -workspace WeexDemo.xcworkspace test -scheme WeexDemo CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO -destination "platform=iOS Simulator,name=iPhone 6"
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -43,6 +45,10 @@ install:
         export PATH=$PATH:$ANDROID_NDK_HOME
         echo "ndk.dir=$ANDROID_NDK_HOME" > android/local.properties
         ;;
+        "ios" )
```
Comment (janpio) [inline .travis.yml]:
```suggestion
      "ios" )
```

File: dangerfile-ios.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+const type_jsfm = 5;
+const type_jsfm_test = 6;
+const type_ui_test = 8;
+
+const getFileType = file => {
+  if (file.match(/WeexSDK\/Sources\/.+\.(m|h|mm)/)) {
+    return type_ios_sdk;
+  } else if (file.match(/WeexSDKTests\//)) {
+    return type_ios_test;
+  } else if (file.match(/android\/sdk\/src\/test\/.+\.java/)) {
```
Comment (janpio) [inline dangerfile-ios.js]: ??

Comment (wrmswindmill) [reply dangerfile-ios.js]: this is a common function of `dangerfile-ios.js`,`dangerfile-android.js` and `dangerfile.js`

Comment (wrmswindmill) [reply dangerfile-ios.js]: I'll change it

Comment (janpio) [reply dangerfile-ios.js]: Was this used by both before? Maybe it can still be a unified file? Otherwise, yes removing the Android parts would make sense.

Comment (wrmswindmill) [reply dangerfile-ios.js]: the `dangerfile-android.js` and `dangerfile-ios.js` should be different in weex-playground project,and there should be not exists `dangerfile.js`. I will update the code.

**Comment by wrmswindmill** [comment]
I'm sorry to test so much times online, because my local programming env is broken.(I am fixing it) and in my Travis CI ,it build success(but actually it should be failed)

**Comment by wrmswindmill** [comment]
Error log:
```
so 'base/third_party/icu/icu_utf.h' file not found
```

this error caused by  the pr in `weex-sdk` :https://github.com/apache/incubator-weex/pull/2693
it change the file `WeexSDK.podspec` at root directory,
```
exclude:
'weex_core/Source/base/message_loop',
'weex_core/Source/base/third_party',
 'weex_core/Source/base/thread',
```

so `base/third_party/icu/icu_utf.h` not found.

**Review by wqyfavor — COMMENTED** [review]
Some xcuserdata file should not be committed.

**Review by wqyfavor — APPROVED** [review]
OK

**Pull request merged**